generate_rospkg_apkbuild/genapkbuild.py

```diff
@@ -266,6 +266,8 @@ def package_to_apkbuild(ros_distro, package_name,
     ament_python = False
 
     build_type = pkg.get_build_type()
+    if not is_ros2 and not pkg.has_buildtool_depend_on_catkin():
+        build_type = 'cmake'
     if build_type == 'catkin':
         catkin = True
     elif build_type == 'cmake':
```

Comment:
Do we still need this condition?

Reply:
It's not that necessary but if there will be automatic updates for ROS 1 packages registered in `aports-ros-experimental`, this change might be useful as we don't need to fix them by hand.

Reply:
I thought L271 is `elif`.
Maybe better to add a newline after L270.

If ROS2 package may specify `<build_type>cmake</build_type>`, this should be required for ROS2 package with `build_type=cmake`.

Reply:
I understand what you mean now. I agree to change L271 to `elif`. Done in https://github.com/alpine-ros/ros-abuild-docker/pull/168/commits/fa0f9b9c6efa4e00abb1e6d668a7fbad384631f3.